source/renderer/app/components/wallet/WalletSendForm.js

```diff
@@ -700,27 +722,51 @@ export default class WalletSendForm extends Component<Props, State> {
       !this.hasAvailableAssets ? styles.disabled : null,
       'primary',
     ]);
+
+    const receiverFieldClasses = classNames([
+      styles.receiverInput,
+      this.hasReceiverValue() &&
```

Comment:
@aleksandardjordjeviciohk this part is repeated in two places - can we extract it into a method and reuse it in both?

Reply:
Fixed